### PR TITLE
remove @author, updated on, created on, and other useless remarks

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

File: code/src/java/pcgen/core/ClassType.java [hunk 1]
```diff
@@ -19,6 +19,11 @@
 
 import pcgen.base.lang.UnreachableError;
 
+
+/**
+ * {@code Campaign}.
```
Comment (thpr) [inline code/src/java/pcgen/core/ClassType.java]: This seems out of place?

Comment (grimreaper) [inline code/src/java/pcgen/core/ClassType.java]: resolved

**Comment by LegacyKing** [comment]
Conflicted

**Comment by grimreaper** [comment]
resolved

**Pull request merged**